.github/workflows/test-coverage.yml

```diff
@@ -274,6 +274,32 @@ jobs:
           coverage-artifact-name: "coverage_telemetry"
           coverage-path: coverage.txt
 
+  # This job runs the database js tests using default configuration, on linux.
+  test-js:
+    name: Test JS job
+
+    runs-on: ubuntu-latest
+
+    env:
+      CGO_ENABLED: 0
+
+    steps:
+      - name: Checkout code into the directory
+        uses: actions/checkout@v4
+
+      - name: Setup defradb
+        uses: ./.github/composites/setup-defradb
+
+      - name: Run integration tests
+        shell: bash # It's required for run step to specify shell in a composite action.
```

Comment:
todo: This is not a composite action so this `shell` is not needed per run (is set at global level I believe).
Please remove

Reply:
Thanks for catching that. I didn't know if it was needed or not.